sway/input/seat.c

```diff
@@ -1380,9 +1380,9 @@ void seat_apply_config(struct sway_seat *seat,
 
 	wl_list_for_each(seat_device, &seat->devices, link) {
 		seat_configure_device(seat, seat_device->input_device);
+		cursor_handle_activity(seat->cursor,
+			seat_device->input_device->wlr_device);
 	}
-
-	cursor_handle_activity(seat->cursor, IDLE_SOURCE_POINTER);
```

Comment:
Note for reviewer: I'm not sure what this call was supposed to accomplish here, so I've just made it trigger for all devices on the seat rather than just the pointer.

Reply:
This was introduced in 4d88c957905e7.

Probably the intent was to reset/restart the idle timers in case their configuration changed?

cc @RedSoxFan

Reply:
emersion is correct on the reasoning. It predated the source argument, which is why it was only called once for the seat.